src/ks-app/src/app/containers/nav/vertical-nav.component.html

```diff
@@ -0,0 +1,201 @@
+<h1>Vertical Nav</h1>
+
+<div class="content-wrapper">
+    <div class="main-container">
+        <header class="header header-6">
+            <div class="branding">
+                <a href="#">
+                    <clr-icon shape="vm-bug"></clr-icon>
+                    <span class="title">Project Clarity</span>
+                </a>
+            </div>
+            <form class="search">
+                <label for="search_input">
+                    <input id="search_input" type="text" placeholder="Search for keywords...">
+                </label>
+            </form>
+            <div class="settings">
+                <a href="javascript://" class="nav-link nav-icon">
+                    <clr-icon shape="cog"></clr-icon>
+                </a>
+            </div>
+        </header>
+        <div class="content-container">
+            <div class="content-area">
+                <div class="example-options">
+                    <h6>Options</h6>
+                    <div class="btn-group">
+                        <div class="radio btn">
+                            <input type="radio" name="vertNavRadios" id="rad1" [(ngModel)]="option" value="link">
+                            <label for="rad1">Nav Group Link</label>
+                        </div>
+                        <div class="radio btn">
+                            <input type="radio" name="vertNavRadios" id="rad2" [(ngModel)]="option" value="text">
+                            <label for="rad2">Nav Group Text</label>
+                        </div>
+                    </div>
+                </div>
+                <div class="controls">
+                    <h6>Controls</h6>
+                    <button class="btn" (click)="toggleNav()">Toggle Nav</button>
+                    <button class="btn" (click)="toggleGroup()">Toggle Group</button>
+                </div>
+
+                <clr-alert [clrAlertType]="'alert-danger'" [clrAlertClosable]="false" *ngIf="option === 'text'">
+                    <div class="alert-item">
+                        <span class="alert-text">
+                            Lazy Loading + Nav Group parent text (instead of parent links)
+                            will not highlight when a child route is active and the nav group is collapsed.
+                            If you don't have a nav group parent link we recommend that you do not use lazy loading.
+                        </span>
+                    </div>
+                </clr-alert>
+
+                <router-outlet></router-outlet>
+            </div>
+            <clr-vertical-nav *ngIf="option === 'link'"
+                              [clrVerticalNavCollapsible]="true"
+                              [clrVerticalNavCollapsed]="navCollapsed"
+                              (clrVerticalNavCollapsedChange)="updateNavCollapsed($event)">
+
+                <clr-vertical-nav-group routerLinkActive="active">
+                    <a
```

Comment:
Vertical Nav demos might have to be replaced with the ones which just use text in nav groups because thats the default vertical nav.

Reply:
Ok. Will do.